rockspec/apisix-dev-1.0-0.rockspec

```diff
@@ -29,6 +29,7 @@ dependencies = {
     "lua-resty-openidc = 1.7.2",
     "opentracing-openresty = 0.1",
     "lua-resty-radixtree = 0.4",
+    "lua-resty-iputils = 0.3.0-1",
```

Comment:
add COPYRIGHT for this library: https://github.com/iresty/apisix/blob/master/COPYRIGHT